ParkingZoneApp/IRepository/IRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ParkingZoneApp.IRepository
+{
+    public interface IRepository<T>
+    {
+        List<T> GetAll();
+        T GetByID(Guid? id);
+        void Add(T value);
+        void Update(T value);
+        T Delete(Guid Id);
```

Comment:
```suggestion
        void Delete(Guid Id);
```

Reply:
should be void because when you delete data, how can you return it after deleting :)? It has already deleted, so we cannot return it